wallet/udb/txdb.go

```diff
@@ -1704,6 +1721,15 @@ func putTxScript(ns walletdb.ReadWriteBucket, script []byte) error {
 	return nil
 }
 
+func removeTxScript(ns walletdb.ReadWriteBucket, script []byte) error {
```

Comment:
keeping with the standard naming in this file, this should be named deleteTxScript.